spec/features/users_index_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  before :each do
+    @user1 = User.create!(name: 'Zia', photo: 'https://unsplash.com/photos/1.jpg', bio: 'Teacher from Mexico.')
+    @user2 = User.create!(name: 'Ali', photo: 'https://unsplash.com/photos/2.jpg', bio: 'Teacher from Mexico.')
+    @user3 = User.create!(name: 'Rashid', photo: 'https://unsplash.com/photos/3.jpg', bio: 'Teacher from Mexico.')
+  end
+
+  describe 'index page' do
+    it 'should display the username of all other users' do
+      visit users_path
+      expect(page).to have_content(@user1.name)
+      expect(page).to have_content(@user2.name)
+      expect(page).to have_content(@user3.name)
+    end
+  end
+end
```

Comment:
- [ ] I noticed you wrote only the first test for this file whereas they are supposed to be 4 in number, kindly finish the test. 
![image](https://github.com/fatmahussein/Blog-app/assets/86200356/bbd4dd81-edf0-415e-84a2-e7b0c1a4ddba)